android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/remote/SyncPerformanceIntTestsHarness.kt

```diff
@@ -170,14 +175,20 @@ class SyncPerformanceIntTestsHarness : BaseStitchAndroidIntTest() {
                     if (testParams.outputToStitch) {
                         val filterDocument = Document("_id", resultId)
                         val updateDocument = Document()
-                            .append("\$push", Document("results", runResult.asBson.toJson()))
+                            .append("\$push", Document("results", runResult.asBson))
+                            .append("\$set", Document("status", "Success"))
                         Tasks.await(outputColl.updateOne(filterDocument, updateDocument))
                     }
                 }
             }
         } catch (e: Exception) {
-            Tasks.await(outputColl.deleteOne(Document("_id", resultId)))
-            throw e
+            Tasks.await(outputColl.updateOne(
+                    Document("_id", resultId),
+                    Document("\$set", Document()
```

Comment:
@tkaye407 I can modify this whenever your approach for iOS is set in stone

Reply:
Ill go in and make some changes to the android harness once Im done with iOS

Reply:
Yeah currently this is not the best behaviour because a failure will stop all tests but I can fix it later=